otx/algorithms/segmentation/adapters/mmseg/task.py

```diff
@@ -0,0 +1,563 @@
+                mmseg_version=__version__ + get_git_hash()[:7],
+                CLASSES=target_classes,
+            )
+
+        # Model
+        model = self.build_model(cfg, fp16=cfg.get("fp16", False))
+        model.train()
+        model.CLASSES = target_classes
+
+        if cfg.distributed:
+            torch.nn.SyncBatchNorm.convert_sync_batchnorm(model)
+            if cfg.dist_params.get("linear_scale_lr", False):
+                new_lr = len(cfg.gpu_ids) * cfg.optimizer.lr
+                logger.info(
+                    f"enabled linear scaling rule to the learning rate. \
+                    changed LR from {cfg.optimizer.lr} to {new_lr}"
+                )
+                cfg.optimizer.lr = new_lr
+
+        validate = bool(cfg.data.get("val", None))
+        train_segmentor(
+            model,
+            datasets,
+            cfg,
+            distributed=cfg.distributed,
+            validate=validate,
+            timestamp=timestamp,
+            meta=meta,
+        )
+
+        # Save outputs
+        output_ckpt_path = os.path.join(cfg.work_dir, "latest.pth")
+        best_ckpt_path = glob.glob(os.path.join(cfg.work_dir, "best_mDice_*.pth"))
+        if len(best_ckpt_path) > 0:
+            output_ckpt_path = best_ckpt_path[0]
+        best_ckpt_path = glob.glob(os.path.join(cfg.work_dir, "best_mIoU_*.pth"))
+        if len(best_ckpt_path) > 0:
+            output_ckpt_path = best_ckpt_path[0]
+        return dict(
+            final_ckpt=output_ckpt_path,
+        )
+
+    def _get_feature_module(self, model):
+        return model
+
+    def _explain_model(self):
+        """Explain function of OTX Segmentation Task."""
+        raise NotImplementedError
+
+    # pylint: disable=too-many-statements
+    def _export_model(
+        self,
+        precision: ModelPrecision = ModelPrecision.FP32,
+        dump_features: bool = True,
+    ):
+        """Export function of OTX Segmentation Task."""
+        # copied from OTX inference_task.py
+        self._init_task(export=True)
+
+        cfg = self.configure(False, "test", None)
+
+        self._precision[0] = precision
+        export_options: Dict[str, Any] = {}
+        export_options["deploy_cfg"] = self._init_deploy_cfg()
+        if export_options.get("precision", None) is None:
+            assert len(self._precision) == 1
+            export_options["precision"] = str(self._precision[0])
+
+        export_options["deploy_cfg"]["dump_features"] = dump_features
+        if dump_features:
+            output_names = export_options["deploy_cfg"]["ir_config"]["output_names"]
+            if "feature_vector" not in output_names:
+                output_names.append("feature_vector")
+            if export_options["deploy_cfg"]["codebase_config"]["task"] != "Segmentation":
+                if "saliency_map" not in output_names:
+                    output_names.append("saliency_map")
+        export_options["model_builder"] = build_segmentor
```

Comment:
Sorry for late update, this is the cause of fake quantization error
```suggestion
        export_options["model_builder"] = getattr(self, "model_builder", build_segmentor)
```

Reply:
Fixed, please review again, thanks!